ocw/lib/gce.py

```diff
@@ -36,9 +38,13 @@ def _paginated(self, api_call, **kwargs) -> list:
 
     def _delete_resource(self, api_call, resource_name, *args, **kwargs) -> None:
         resource_type = {
-            self.compute_client().instances: "instance",
-            self.compute_client().images: "image",
             self.compute_client().disks: "disk",
+            self.compute_client().firewalls: "firewall",
```

Comment:
does the order matter here?

Reply:
Does not some resources to be deleted before than others as they are inter-dependent ?

Reply:
This is a dictionary to get the resource type name from the API method.

We clean up the resources in `cleanup_all()` in order.  Check the force-pushed commit.